From 7314d232f8e85879d8f4c311ced44ee5b21fb239 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Wed, 10 Feb 2016 23:26:27 +0100
Subject: [PATCH] Makefile: allow to override the PREFIX variable

Some people may not want to install in /usr/local, so this commit
makes it possible to override the PREFIX variable from the make
command line.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---

 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 69f5aca..f9a6781 100644
--- a/Makefile
+++ b/Makefile
@@ -20,7 +20,7 @@
 # $Id$
 
 DESTDIR=
-PREFIX=/usr/local
+PREFIX?=/usr/local
 ETCDIR=/etc/vpnc
 BINDIR=$(PREFIX)/bin
 SBINDIR=$(PREFIX)/sbin
-- 
2.6.4

